docs/DeveloperGuide.md

```diff
@@ -1075,7 +1075,44 @@ testers are expected to do more *exploratory* testing.
     1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
        Expected: Similar to previous.
 
-1. _{ more test cases …​ }_
+### Showing search terms
+
+1. Showing unique applicant names when multiple applicants exist in the list
+
+   1. Prerequisites: Multiple applicants in the list.
+
+   2. Test case: `show n/`<br>
+      Expected: Result display shows a list of unique names belonging to your list of **all** applicants.
+
+   3. Test case: `show n/John`<br>
+      Expected: `John` input for name prefix `n/` is ignored. Result display shows a list of unique names belonging
+      to your list of **all** applicants.
+
+   4. Test case: `show`<br>
+      Expected: No search terms are shown on result display. Error details shown in the result display.
```

Comment:
Since this category is 'showing unique applicant names when...', this command seem not to be relatable to applicants' names. Maybe you can put this test case in a new category e.g. 'showing without prefixes'

Reply:
Thanks for the suggestion. I agree that it is not relatable to applicant's name and will make the necessary changes.